sdk/scheduler/build.gradle

```diff
@@ -139,7 +139,7 @@ dependencies {
     compile "javax.el:javax.el-api:${elVer}"
     compile "org.glassfish.web:javax.el:${elVer}"
     compile 'com.mesosphere:mesos-http-adapter:0.4.1'
-    compile 'org.apache.mesos:mesos:1.4.0-rc1'
+    compile 'org.apache.mesos:mesos:1.5.0'
```

Comment:
Could make sense to have a comment here mentioning that this corresponds to the version of Mesos in DC/OS 1.11.